agents/ppo.py

```diff
@@ -97,11 +97,11 @@ def forward(self, inputs: Dict[str, Any]) -> Dict[str, Any]:
                 bag_ids=bag_ids,
                 node_idxs=current_node_idx,
                 extra_infos=zip(
-                    current_state_value_function,
-                    neighbors_logits,
+                    current_state_value_function.detach(),
```

Comment:
Wait, wait, wait. What do you need  `detach` here? I thought you want to store all the information about gradient flows as well.

Reply:
If this works bad, will be fixed further